arithmetization/src/test/java/net/consensys/linea/replaytests/Issue1031Tests.java

```diff
@@ -28,6 +28,6 @@ public class Issue1031Tests {
 
   @Test
   void issue_1090_range_10_20() {
-    replay(LINEA_MAINNET, "10-20.mainnet.json.gz");
+    replay(OLD_LINEA_MAINNET, "10-20.mainnet.json.gz");
```

Comment:
Was the chain id (I take it) of mainnet changed ?

Reply:
@OlivierBBB no, the chain id is unchanged for linea mainnet.  What has changed, however, is the block gas limit constraint.  All of our existing replay tests were generated on a version of Linea which didn't have this restriction.  @letypequividelespoubelles called this "old linea" and I've just borrowed that.  Using the chain configuration `OLD_LINEA_MAINNET` here means that the `zkevm.bin` will be run with the constant `GAS_LIMIT_ENABLE=0` and, as a result, the constraint `block-gas-limit-value` will have effect (see [here](https://github.com/Consensys/linea-constraints/blob/master/blockdata/processing/gaslimit/linea.lisp))